File: snippets/fsharp/System/Decimal/op_Addition/lessthanorequal1.fs
module lessthanorequal1

// <Snippet17>
open System

let number1 = 16354.0699m
let number2 = 16354.0695m
printfn $"{number1} <= {number2}: {number1 <= number2}"

let rounded1 = Decimal.Round(number1, 2)
let rounded2 = Decimal.Round(number2, 2)
printfn $"{rounded1} <= {rounded2}: {rounded1 <= rounded2}"

// The example displays the following output:
//       16354.0699 <= 16354.0695: False
//       16354.07 <= 16354.07: True
// </Snippet17> 

File: snippets/csharp/System/Decimal/op_Addition/lessthanorequal1.cs
// <Snippet17>
using System;

public class Example
{
    public static void Main()
    {
        decimal number1 = 16354.0699m;
        decimal number2 = 16354.0695m;
        Console.WriteLine($"{number1} <= {number2}: {number1 <= number2}");

        number1 = decimal.Round(number1, 2);
        number2 = decimal.Round(number2, 2);
        Console.WriteLine($"{number1} <= {number2}: {number1 <= number2}");
    }
}
// The example displays the following output:
//       16354.0699 <= 16354.0695: False
//       16354.07 <= 16354.07: True
// </Snippet17>
